OpenRA.Mods.Common/Widgets/Logic/IntroductionPromptLogic.cs

```diff
@@ -43,8 +44,18 @@ public IntroductionPromptLogic(Widget widget, ModData modData, WorldRenderer wor
 			var graphicSettings = modData.GetSettings<GraphicSettings>();
 			var gameSettings = modData.GetSettings<GameSettings>();
 
-			classic = FluentProvider.GetMessage(Classic);
-			modern = FluentProvider.GetMessage(Modern);
+			var controlTypes = new Dictionary<MouseControlStyle, string>
+			{
+				{ MouseControlStyle.Classic, FluentProvider.GetMessage(Classic) },
+				{ MouseControlStyle.Modern, FluentProvider.GetMessage(Modern) },
+				{ MouseControlStyle.OtherRTS, FluentProvider.GetMessage(OtherRTS) },
+			};
+
+			// TODO: Have this check gs.Yaml.NodeWithKeyOrDefault("UseClassicMouseStyle") when IntroductionPromptVersion < 2
+			// Requires PR #22219 to be merged
```

Comment:
This has now been merged. If you change `IntroductionVersion` back to 1 I could implement and migrate the settings from #22253 in its own PR.

Reply:
This doesn't work at all if InputSettingsLogic.ShowMouseControlDropdown is non-static, since that's used here too